RUFAS/weather.py

```diff
@@ -93,14 +92,17 @@ def __init__(self, weather_file: dict, time: Time):
             dict(info_map, **{"units": MeasurementUnits.DEGREES_CELSIUS}),
         )
 
-    def get_current_day_conditions(self, time: Time) -> CurrentDayConditions:
+    def get_current_day_conditions(self, time: Time, latitude: float | None = None) -> CurrentDayConditions:
         """
         Creates a CurrentDayConditions object containing all the weather conditions on the current day.
 
         Parameters
         ----------
         time: Time
             Time object containing the current time of the simulation.
+        latitude : float | None, default None
```

Comment:
```suggestion
        latitude : float | None, default=None
```
Very small thing here and not necessary to implement but it seems like some places we have `default=xyz` and other places we're doing `default xyz`. Seems like Numpy counts them as equivalent so only need to decide if we care that they all look the same internally.

Reply:
I have noticed the same thing. Since Numpy seems ambivalent about how its done, I think it isn't a big deal if it is inconsistent in RuFaS. Though it would be nice to have them all be the same.